omnibus/package-scripts/agent-deb/postinst

```diff
@@ -65,35 +67,24 @@ install_method:
     echo "$install_info_content" > $CONFIG_DIR/install_info
 fi
 
-# Set proper rights to the dd-agent user
-chown -R dd-agent:dd-agent ${CONFIG_DIR}
-chown -R dd-agent:dd-agent ${LOG_DIR}
-chown -R dd-agent:dd-agent ${INSTALL_DIR}
+# Set proper rights to the dd-agent user and group
+chown -R root:dd-agent ${CONFIG_DIR}
```

Comment:
This will also require changes in some other repos as well, otherwise it won't be effective for a lot of customers:

* https://github.com/DataDog/agent-linux-install-script/blob/main/install_script.sh.template#L693
* https://github.com/DataDog/ansible-datadog/blob/main/tasks/_agent-linux-macos-shared.yml
* https://github.com/DataDog/chef-datadog/blob/main/recipes/_agent6_config.rb#L54
* https://github.com/DataDog/puppet-datadog-agent/blob/main/manifests/init.pp#L542
* https://github.com/DataDog/datadog-formula/blob/main/datadog/config.sls

Reply:
:+1: